src2/App.css

```
@@ -0,0 +1,44 @@
+.App {
+  text-align: center;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  padding: 0;
+  font-family: "ヒラギノ角ゴシック Pro", "Hiragino Kaku Gothic Pro", "メイリオ",
+    "Meiryo, Osaka", "ＭＳ Ｐゴシック", "MS PGothic", "sans-serif" !important;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
```

```suggestion
.App-logo {
  height: 20vmin;
  pointer-events: yes;
}
```
40 is pretty bad, can you change it like this ?

---

```suggestion
.App-logo {
  height: 40vmin;
  pointer-events: none;
}
```
Is usefull for this project ?

---

okay, im sorry